cssom-1/Overview.bs

```diff
@@ -2779,19 +2782,27 @@ steps:
   versions. That is, both <code>:before</code> and <code>::before</code> should
   match above.
 
+ <li>Let <var>decls</var> be an empty list of <a>CSS declarations</a>.
+ <li>If <var>elt</var> is <a>connected</a>, part of the <a>flat tree</a>,
+  and its <a>shadow-including root</a>'s <a>browsing context</a> either doesn't have a
```

Comment:
What if the root doesn't have a browsing context?

Reply:
Could you put an example of a document which has no browsing context? It's not clear to me when can that happen.

Reply:
This is why I asked you for review btw, thanks a lot for taking a look :-)

Reply:
`new Document()`, `xhr.responseXML`, `new DOMParser().parseFromString(...)`, ...

Reply:
Alright, I think we don't want to return styles there either, since those documents have no rendering behavior. Pushed two fixups for the issues you raised, mind taking a look? Thanks!